From ee349175fe9d1d0d22097f05ebc9b150d452a884 Mon Sep 17 00:00:00 2001
From: "vicent.chi" <vicent.chi@rock-chips.com>
Date: Tue, 16 Jun 2020 17:59:37 +0800
Subject: [PATCH] add install libRxcpp.so command

---
 projects/CMake/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/projects/CMake/CMakeLists.txt b/projects/CMake/CMakeLists.txt
index 542a91a..bd5d614 100644
--- a/projects/CMake/CMakeLists.txt
+++ b/projects/CMake/CMakeLists.txt
@@ -1,3 +1,4 @@
+w
 cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
 
 project(rxcpp VERSION 2.2.4 LANGUAGES C CXX)
@@ -135,5 +136,6 @@ SET_TARGET_PROPERTIES(RxCpp PROPERTIES LINKER_LANGUAGE CXX)
 
 set(CMAKE_SKIP_INSTALL_ALL_DEPENDENCY TRUE CACHE BOOL "Don't require all projects to be built in order to install" FORCE)
 
+install(TARGETS RxCpp LIBRARY DESTINATION lib)
 install(DIRECTORY ${RXCPP_DIR}/Rx/v2/src/rxcpp/ DESTINATION include/rxcpp
         FILES_MATCHING PATTERN "*.hpp")
-- 
2.26.1

